examples/booking_app/lib/feature_hotel/search/blocs/hotel_list_bloc.dart

```diff
@@ -105,29 +84,27 @@ class HotelListBloc extends $HotelListBloc {
     ),
   );
 
-  final _searchFilterSubject =
-      BehaviorSubject<HotelSearchFilters>.seeded(HotelSearchFilters());
-  final _querySubject = BehaviorSubject<String>.seeded('');
-  final _dateRangeSubject = BehaviorSubject<DateTimeRange?>.seeded(null);
-
   @override
   Stream<PaginatedList<Hotel>> get searchedHotels => _hotels;
 
   @override
   void dispose() {
-    _searchFilterSubject.close();
-    _querySubject.close();
-    _dateRangeSubject.close();
     _hotels.close();
     super.dispose();
   }
 
   @override
   Stream<String> get query =>
-      _searchFilters.map((filters) => filters.query).shareReplay(maxSize: 1);
+      filters.map((filters) => filters.query).shareReplay(maxSize: 1);
 
   @override
-  Stream<HotelSearchFilters> get filters => _searchFilters;
+  Stream<HotelSearchFilters> get filters => Rx.combineLatest2(
```

Comment:
@DDavidPrime do we need to expose this property, or we can keep it private ?

Reply:
This state is used for rebuilding the Date range widget. In future, when implementing other types of filtering, this state can be used for providing the data, instead of multiple different states to the same widget (instead of nesting multiple RxBuilder methods or similar).